tests/integration/ha_tests/test_self_healing.py

```diff
@@ -116,9 +114,9 @@ async def test_kill_db_process(
     ), "secondary not up to date with the cluster after restarting."
 
 
-@pytest.mark.parametrize("process", DB_PROCESSES)
+@pytest.mark.parametrize("process", [PATRONI_PROCESS])
```

Comment:
Pkill was likely freezing the wrong process before. I was unable to get patroni to switch over when testing manually.